core/app/models/spree/promotion/rules/item_total.rb

```diff
@@ -36,12 +36,12 @@ def eligible?(order, _options = {})
         private
 
         def formatted_amount_min
```

Comment:
```suggestion
        def formatted_amount_min(order)
```

Reply:
@jhovad you also need to pass the order from `eligable?` to `ineligible_message_max` / `ineligible_message_min`